src/__test__/dataparitytests/epochs.parity.test.ts

```diff
@@ -0,0 +1,170 @@
+import gql from 'graphql-tag'
+import { TestClient } from '../TestClient'
+import fetch from 'node-fetch'
+
+
+async function getDataFromAPI (path: string) {
+  const response = await fetch(`https://explorer.cardano.org/api/${path}`)
+  return response.json()
+}
+
+export function epochsTests (createClient: () => Promise<TestClient>) {
+  describe('epochs', () => {
+    let client: TestClient
+    let restData: any
+    let graphQLData: any
+    const epoch = 111
+    const slot = 3313
+
+    beforeAll(async () => {
+      client = await createClient()
+      const restResult = await getDataFromAPI(`epochs/${epoch}/${slot}`)
+      restData = restResult.Right[0]
+      const graphQLResult = await client.query({
+        query: gql`query EpochDetails{
+			epochs(where: {number:{_eq:${epoch}}}){
+                            number
+                            startedAt
+                            lastBlockTime
+			    blocks (where: {slotWithinEpoch:{_eq:${slot}}}){
+                              slotNo
+                              epochNo
+                              slotWithinEpoch
+                              number
+                              id
+                              createdAt
+                              transactionsCount
+                              transactions{
+                                totalOutput
+                              }
+                              size
+                              createdBy
+                              fees
+                            }
+                            output
+                          }
+                        }`
+      })
+      graphQLData = graphQLResult.data.epochs[0].blocks[0]
+    }, 60000)
+
+    it('Returns the same epoch number', async () => {
+
+      const restResultEpochNumber = restData.cbeEpoch
+      const graphQLEpochNumber = graphQLData.epochNo
+
+      console.log("rest epoch no: " + restResultEpochNumber)
```

Comment:
I will remove all these `console.log` lines when this will move from draft to ready state.